docs/METADATA.md

```diff
@@ -64,7 +64,7 @@ of delegated Targets metadata and [example](https://raw.githubusercontent.com/th
 
 Signed by: Snapshot role.
 
-The snapshot.json metadata file lists hashes and sizes of all metadata files other than timestamp.json. This file ensures that clients will see a consistent view of the files on the repository. That is, metadata files (and thus target file) that existed on the repository at different times cannot be combined and presented to clients by an attacker.
+The snapshot.json metadata file lists the version, and optionally the file hashes and sizes, of all metadata files other than root.json and timestamp.json. This file ensures that clients will see a consistent view of the files on the repository. That is, metadata files (and thus target file) that existed on the repository at different times cannot be combined and presented to clients by an attacker.
```

Comment:
Minor nit: The *"all metadata files other"* wording is absolutely correct, but wouldn't it be more helpful to tell the reader what files to include, i.e. *"top-level targets metadata and all delegated targets metadata"* (instead of what files not to include)?
What do others think? This is definitely not a blocker for this PR. 

I proposed a similar change in the spec (see https://github.com/theupdateframework/specification/commit/8dc300f3d24298d3ccc3d154cd6221ed3f22843b and the newer https://github.com/theupdateframework/specification/commit/a5848e758a7aab27cb7143ecec287251a4f6d382)

Reply:
That would be much more helpful indeed. I've force-pushed an update to this patch which changes this to:
> The snapshot.json metadata file lists the version, and optionally the file hashes and sizes, of the top-level targets metadata and all delegated targets metadata.